resources/web/wwi/nodes/WbWorld.js

```diff
@@ -0,0 +1,22 @@
+export default class WbWorld {
+  constructor() {
+    this.hasFog = false;
+    this.coordinateSystem = 'ENU';
+    this.upVector = glm.vec3(0, 1, 0);
```

Comment:
```suggestion
    this.upVector = glm.vec3(0, 0, 1);
```
Should it be like this?